app.js

```diff
@@ -9,6 +9,77 @@ const bar3 = document.querySelector('.bar:nth-child(3)');
 
 const navMenu = document.querySelector('.nav-menu');
 const xIcon = document.querySelector('.x-icon');
+const works = document.querySelector('#works');
+const popupView = document.querySelector('.popup-view-bg');
+
+const projects = [
+  {
+    id: '1',
+    title: 'Tonic',
+    image: 'images/section-2.png',
```

Comment:
- [ ] Kindly add some padding around the pop up image in desktop view so the design looks more like the Figma design

Your design
![image](https://user-images.githubusercontent.com/81932478/180201880-a2414b7a-1151-4002-be3c-84bff0d9f86e.png)

Provided design
![image](https://user-images.githubusercontent.com/81932478/180202003-1bba6fa7-f0f3-48c4-b2e6-1de92c5de749.png)

- [ ] Kindly add some margin-bottom to your mobile pop-up version as well so that your design matches the provided layout.

Your design
![image](https://user-images.githubusercontent.com/81932478/180202461-a04023e6-01ff-44ef-988b-b2ce26c89e0e.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/180202581-10136968-86ac-4d88-87a7-c46854d66910.png)